contrib/release-dokku

```diff
@@ -415,6 +417,8 @@ main() {
     git checkout -- plugins/*/plugin.toml
   fi
 
+  # what about this?
+  # echo "build/dokku_${NEXT_VERSION}_arm64.deb" >build/deb-filename 
```

Comment:
Drop these two lines.

Reply:
What's the idea behind those other two filename lines?

Reply:
They are used by tests to extract the correct file. The tests run in amd64.